<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sodapop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>271</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="summaryLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Here's a summary of your population</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="summaryBrowser">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>39</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>261</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Welcome to Sodapop!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>526</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Run analysis utilities</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Run simulation</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="simParameters">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>39</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Simulation parameters</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="geneListButton">
       <property name="text">
        <string>Load gene list</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="noGen">
       <property name="text">
        <string>Number of generations</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="summButton">
       <property name="text">
        <string>Load snapshot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="simName">
       <property name="text">
        <string>Simulation name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="simSize">
       <property name="text">
        <string>Simulation size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="snapInterval">
       <property name="text">
        <string>Snapshot interval</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sizeEdit">
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="genEdit">
       <property name="text">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="snapEdit">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="summPath"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="genelistPath"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="fitnessFunction">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>279</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitness function</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>89</y>
       <width>501</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="DFE">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Distribution of fitness effects</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>19</y>
           <width>231</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <widget class="QComboBox" name="distributionType">
            <property name="insertPolicy">
             <enum>QComboBox::InsertAtBottom</enum>
            </property>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gamma</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Exponential</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Shape</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Scale</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="shapeEdit"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="scaleEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="fitLandscape">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Fitness landscape</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>241</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QLineEdit" name="landscapeAEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="landscapeB">
            <property name="text">
             <string>Load matrix</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="landscapeA">
            <property name="text">
             <string>Load matrix</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="landscapeBEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="theoryOrExp">
      <item>
       <widget class="QRadioButton" name="theoretical">
        <property name="text">
         <string>Theoretical</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">functionTypeGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="experimental">
        <property name="text">
         <string>Experimental</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">functionTypeGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="functionType">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>501</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Additive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Multiplicative</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Neutral</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>781</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="chooseWorkDir">
       <property name="text">
        <string>Select working directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workingDir"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuCreate">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionSnapshot"/>
     <addaction name="actionSummary"/>
     <addaction name="actionCell"/>
     <addaction name="actionGene_list"/>
     <addaction name="actionGene"/>
    </widget>
    <addaction name="menuCreate"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSummary">
   <property name="text">
    <string>Population summary</string>
   </property>
  </action>
  <action name="actionCell">
   <property name="text">
    <string>Cell</string>
   </property>
  </action>
  <action name="actionGene_list">
   <property name="text">
    <string>Gene list</string>
   </property>
  </action>
  <action name="actionGene">
   <property name="text">
    <string>Gene</string>
   </property>
  </action>
  <action name="actionSnapshot">
   <property name="text">
    <string>Snapshot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="functionTypeGroup"/>
 </buttongroups>
</ui>
